File: Modul_3/Soal_4/PRAK304-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main(void) {
    int angka;
    scanf("%d", &angka);

    if(angka == 0) {
        printf("Nol");
    }
    else if(angka < 10) {
        printf("Satuan");
    }
    else if(angka < 20) {
        printf("Belasan");
    }
    else if(angka < 100) {
        printf("Puluhan");
    }
    else if(angka >= 100) {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}

File: Modul_3/Soal_4/PRAK304-2310817210015-MuhammadBukhariFitri.py
angka = int(input())

if angka == 0:
    print("Nol")
elif angka < 10:
    print("Satuan")
elif angka < 20:
    print("Belasan")
elif angka < 100:
    print("Puluhan")
elif angka >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")
